bundles/framework/divmanazer/component/Chart.js

```diff
@@ -254,7 +268,7 @@ Oskari.clazz.define('Oskari.userinterface.component.Chart', function () {
      * @param { Object } options keys: colors -> color scale, valueRenderer -> function for rendering bar values
      */
     createBarChart: function (data, options) {
-        if (data != undefined && this.svg === null) {
+        if (data !== undefined && this.svg === null) {
```

Comment:
Changing the equals check to strict, will allow `null` value to pass to `handleData()`. Maybe this is now what we want?